scrapyd_k8s.sample-docker.conf

```diff
@@ -16,6 +16,9 @@ repository   = scrapyd_k8s.repository.Local
 # Since this is the Docker example, we choose Docker here.
 launcher     = scrapyd_k8s.launcher.Docker
 
+# Maximum number of jobs running in parallel
+max_proc     = 1
+
```

Comment:
not sure yet if we want to enable this by default
do you know what scrapyd has in its default configuration?

Reply:
In scrapyd the max_proc parameter is not default, so it can be set. I think there are reasons to consider making it default because it enhances cluster stability but on the other hand, if a user did not think of optimal resource usage this batch limiting can lead to a smaller output at a given time. So it's beneficial if we need to be conscious about recourses of the prod setting but not very handy if we want to go all in and extract as much data as possible and faster.

Reply:
Thanks for your thoughts. I think we'd like the default to be the minimum needed, while still having the important elements (e.g. resource limits are important for scheduling). In a cluster there are already limits, so I think it should be fine without. Similarity to scrapyd is also a plus, so I'd favour not having this set by default.

Reply:
Both implementations are now optional and are activated if a user provided max_proc in the scrapyd section, it is also described in details in CONFIG.md